client/packages/coldchain/src/Monitoring/api/TemperatureBreach/api.ts

```diff
@@ -10,22 +11,40 @@ export type ListParams = {
   first: number;
   offset: number;
   sortBy: SortBy<TemperatureBreachFragment>;
-  filterBy: TemperatureBreachFilterInput | null;
+  filterBy:
+    | (TemperatureBreachFilterInput & { datetime?: DatetimeFilterInput })
+    | null;
 };
 
 export const getTemperatureBreachQueries = (sdk: Sdk, storeId: string) => ({
   get: {
     list:
       ({ first, offset, sortBy, filterBy }: ListParams) =>
       async () => {
+        const key =
+          sortBy.key === 'datetime' || sortBy.key === 'temperature'
+            ? TemperatureBreachSortFieldInput.StartDatetime
+            : (sortBy.key as TemperatureBreachSortFieldInput);
```

Comment:
Hey @mark-prins ,

I'm wondering do you need to worry about `unacknowledged` becoming the sort parameter?

Reply:
no - I went through all of the fields that you can sort on both the log and breach tables. there's temperature and datetime/startdatetime/enddatetime only